src/H5Fsuper.c

```diff
@@ -49,6 +49,7 @@
 /********************/
 static herr_t H5F__super_ext_create(H5F_t *f, H5O_loc_t *ext_ptr);
 static herr_t H5F__update_super_ext_driver_msg(H5F_t *f);
+static herr_t H5O__fsinfo_set_version(H5F_t *f, H5O_fsinfo_t *fsinfo);
```

Comment:
This looks like a bit of leftover code.  (Definitely doesn't belong here :-) )

Reply:
It looks like this has moved to H5Oprivate.h -- I will include the header instead.

Reply:
Not _strictly_ necessary, since H5Oprivate.h was being included by H5SMprivate.h, but since this module makes H5O calls directly, it's probably for the best.